Vates/VatesAPI/inc/MantidVatesAPI/PresenterFactories.h

```diff
@@ -13,15 +13,15 @@ class WorkspaceProvider;
 
 class DLLExport EmptyWorkspaceNamePolicy {
 protected:
-  std::string getWorkspaceName(Mantid::API::IMDWorkspace_sptr) {
-    return "__EmptyWorkspaceNamePolicy";
-  }
+  const std::string &
+  getWorkspaceName(const Mantid::API::IMDWorkspace & /*workspace*/);
```

Comment:
@mantidproject/tech-steering How should we mark unused arguments? Do we have or want to set a standard? 

FWIW I prefer commenting out the name vs. using the `UNUSED_ARG` macro or leaving it blank.